cmd/pke/app/phases/kubeadm/version/version.go

```diff
@@ -32,7 +32,7 @@ const (
 	use   = "kubernetes-version"
 	short = "Check Kubernetes version is supported or not"
 
-	constraint = "1.15.x-0 || 1.16.x-0 || 1.17.x-0 || 1.18.x-0"
+	constraint = "1.15.x-0 || 1.16.x-0 || 1.17.x-0 || 1.18.x-0 || 1.19.x-0"
```

Comment:
We should handle depreciation somehow on the long term. We might start with a warning for oldest once-supported versions. (Different issue.)

Reply:
See #115 

So far we simply removed old versions. I agree that's not necessarily ideal.